### PR TITLE
Add a widget

### PR DESCRIPTION
**Pull request by joeljfischer**
* Adds a basic widget
* Adds group boxes to main UI

### CODE REVIEW

File: MobileWeather/APIKey.swift [hunk 1]
```diff
@@ -9,5 +9,5 @@
 import Foundation
 
 struct APIKey {
-    static let apiKey = ""
+    static let apiKey = "d6f1f15f251280a3908a846151e9e61c"
```
Comment (FrankElias77) [inline MobileWeather/APIKey.swift]: I think you forgot to remove your apiKey

File: MobileWeather/DailyConditionsView.swift [hunk 2]
```diff
@@ -10,16 +10,17 @@ import SwiftUI
 
 struct DailyConditionsView: View {
     var dailyForecast: [DailyForecast]
-
+
```
Comment (FrankElias77) [inline MobileWeather/DailyConditionsView.swift]:
```suggestion

```

File: MobileWeather/DailyConditionsView.swift [hunk 3]
```diff
@@ -29,7 +30,7 @@ struct DailyConditionsView_Previews: PreviewProvider {
         let json = Bundle.main.url(forResource: "weather-api-response", withExtension: "json")!
         let jsonData = try! Data(contentsOf: json)
         let data = try! JSONDecoder().decode(WeatherData.self, from: jsonData)
-
+
```
Comment (FrankElias77) [inline MobileWeather/DailyConditionsView.swift]:
```suggestion

```

File: SDL Weather Widget/Info.plist [hunk 4]
```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSExtension</key>
+	<dict>
+		<key>NSExtensionPointIdentifier</key>
+		<string>com.apple.widgetkit-extension</string>
+	</dict>
+	<key>NSWidgetWantsLocation</key>
```
Comment (FrankElias77) [inline SDL Weather Widget/Info.plist]:
```suggestion
        <key>NSLocationWhenInUseUsageDescription</key>
        <string>SDL Weather uses your location to find the weather where you are</string>
	<key>NSWidgetWantsLocation</key>
```

**Review by FrankElias77 — CHANGES_REQUESTED** [review]

File: MobileWeather/APIKey.swift (hunk 1 above)
Comment (joeljfischer) [reply MobileWeather/APIKey.swift]: Fixed and deactivated

**Review by FrankElias77 — APPROVED** [review]

**Pull request merged**